xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/AbstractIT.java

```diff
@@ -47,14 +45,15 @@
 import java.time.Instant;
 import java.util.List;
 import java.util.Objects;
+import java.util.concurrent.TimeUnit;
 import java.util.function.Predicate;
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
 public abstract class AbstractIT {
 
   public static final String SUCCESS_STATUS = "tesSUCCESS";
-  public static final Duration POLL_INTERVAL = Duration.ONE_HUNDRED_MILLISECONDS;
+  public static final long POLL_INTERVAL = 100;
```

Comment:
We should either remove all constants here, or create better constants that use `Duration`. Looking at the usages below, I vote that we delete this constant, for improved test clarity below.

See my suggestions down below.